### PR TITLE
[DOC-11624] Put back inadvertently-removed details about getting the IAM role / service account ID for CMEK

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-11624] Put back inadvertently-removed details about getting the IAM role / service account ID for CMEK

### CODE REVIEW

File: src/current/cockroachcloud/managing-cmek.md [hunk 1]
```diff
@@ -27,6 +27,79 @@ This section shows how to enable CMEK on a CockroachDB {{ site.data.products.adv
...
+      - `regions`/`name`: one entry for each of the cluster's regions. CMEK must be configured in each of a cluster's regions.
+
+      ```json
+      {
+        "id": "blahblahblah-9ebd-43d9-8f42-589c9e6fc081",
+        "name": "docs-rule",
+        "cockroach_version": "v22.1.1",
+        "plan": "DEDICATED",
+        "cloud_provider": "GCP",
+        "account_id": "docs-rule-123",
```
Comment (adwittumuluri) [inline src/current/cockroachcloud/managing-cmek.md:74]: I think it would be worth it to have an actual example, like `crl-prod-xyz`

**Review by adwittumuluri — CHANGES_REQUESTED** [review]
Did we used to have docs that spelled out the service account that the customer should use? I feel like there was _something_ that said "use `crl-kms-user-{Last 12 digits of cluster UUID}@{account_id}.iam.gserviceaccount.com`"

**Comment by adwittumuluri** [comment]
> Did we used to have docs that spelled out the service account that the customer should use?

Found it on [the wayback machine](https://web.archive.org/web/20231002145521/https://www.cockroachlabs.com/docs/cockroachcloud/cmek-ops-gcp#step-2-provision-the-cross-tenant-service-account) 😆

File: src/current/cockroachcloud/managing-cmek.md (hunk 1 above)
Comment (mdlinville) [reply src/current/cockroachcloud/managing-cmek.md]: Done

**Comment by adwittumuluri** [comment]
I think [this section](https://github.com/cockroachdb/docs/blob/6c8e21ad0c2b43c4140a3e73c0b9bf7f95f7d9d1/src/current/cockroachcloud/managing-cmek.md?plain=1#L136-L164) will need to change to mirror what we used to have [here](https://web.archive.org/web/20231002145521/https://www.cockroachlabs.com/docs/cockroachcloud/cmek-ops-gcp#step-2-provision-the-cross-tenant-service-account) for GCP. Namely, we need to tell the customer somewhere to use `crl-kms-user-{CLUSTER_ID}@{PROJECT_ID}.iam.gserviceaccount.com`. I don't think we do that anywhere right now.

**Comment by mdlinville** [comment]
> I think [this section](https://github.com/cockroachdb/docs/blob/6c8e21ad0c2b43c4140a3e73c0b9bf7f95f7d9d1/src/current/cockroachcloud/managing-cmek.md?plain=1#L136-L164) will need to change to mirror what we used to have [here](https://web.archive.org/web/20231002145521/https://www.cockroachlabs.com/docs/cockroachcloud/cmek-ops-gcp#step-2-provision-the-cross-tenant-service-account) for GCP. Namely, we need to tell the customer somewhere to use `crl-kms-user-{CLUSTER_ID}@{PROJECT_ID}.iam.gserviceaccount.com`. I don't think we do that anywhere right now.

Done, PTAL

**Review by adwittumuluri — APPROVED** [review]
Thanks for putting it back!

**Pull request merged**